03_backend_developement/03_testing_our_code/03_going_further_with_tests/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Tests_Exercises",
+  "version": "1.0.0",
+  "description": "",
+  "main": "codeTesting.js",
+  "scripts": {
+    "test": "eslint . && jest"
```

Comment:
Here, you don't need to ignore files for ESLint but you should run the tests to see what happens. ESlint is not happy with the skeletton file.
There are legitimate ESLint errors that will be resolved by writing the tests like `'mockArray' is assigned a value but never used` but others are not.

For instance, you should add `/* global beforeEeach */` at the start of the file to avoid complaints.

Reply:
I made changes, the only error remaining are the `value is assigned but never used`👌🏻